tools/WinObjC.Logging/WinObjC.Logging.targets

```diff
@@ -8,25 +8,57 @@
          NOTE: It is important that this information is added at the end in the .targets instead of in the .props so that consumers don't
          accidentally stomp it. This is contrary to a configurable type property that a consumer may want / need to adjust. To depend on
          this package means the consumer gets these these things.  -->
+    <ItemGroup Condition="'$(ConfigurationType)' != 'StaticLibrary'">
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>    
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.winmd"/>
+
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.winmd"/>
+
+      <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.dll"/>    
+      <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.dll"/>
+
+      <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\*.dll"/>
```

Comment:
>_LibsFromPackage [](start = 7, length = 16)

Nit: LibsFromPackage seems misleading since these are dll's.
Not that big a deal though.